src/Tizen.NUI.Components/Controls/RadioButtonGroup.cs

```diff
@@ -85,5 +129,64 @@ public void Remove(RadioButton radio)
             base.RemoveSelection(radio);
             radio.ItemGroup = null;
         }
+
+        private static void OnIsGroupHolderChanged(Binding.BindableObject bindable, object oldValue, object newValue)
+        {
+            var view = bindable as View;
+
+            if (view == null) return;
+
+            if (!(bool)newValue)
+            {
+                view.SetValue(RadioButtonGroupProperty, null, false, true);
+                return;
+            }
+
+            if (view.GetValue(RadioButtonGroupProperty) == null)
+            {
+                view.SetValue(RadioButtonGroupProperty, new RadioButtonGroup(), false, true);
+            }
+        }
+
+        private static void OnRadioButtonGroupChanged(Binding.BindableObject bindable, object oldValue, object newValue)
+        {
+            var view = bindable as View;
+            if (view == null) return;
+
+            var oldGroup = oldValue as RadioButtonGroup;
+            var newGroup = newValue as RadioButtonGroup;
+
+            if (oldGroup != null)
+            {
+                view.ChildAdded -= oldGroup.OnChildChanged;
+                view.ChildRemoved -= oldGroup.OnChildChanged;
+                oldGroup.RemoveAll();
+            }
+
+            if (newGroup != null)
+            {
+                view.ChildAdded += newGroup.OnChildChanged;
+                view.ChildRemoved += newGroup.OnChildChanged;
+                newGroup.OnChildChanged(view, null);
+            }
+        }
+
+        private void OnChildChanged(object sender, EventArgs args)
+        {
+            if (sender is View view)
+            {
+                RemoveAll();
+                foreach (var child in view.Children) if (child is RadioButton radioButton) Add(radioButton);
```

Comment:
Is it OK in NUI convention? (I don't know exactly).

Reply:
Fixed!